onnx_xla/backend.cc

```diff
@@ -101,6 +101,28 @@ namespace onnx_xla {
     #undef OPERATION
   }
 
+  std::unique_ptr<Literal> XlaExecutor::descriptorToLiteral(const onnxTensorDescriptor& t) {
+    std::vector<int64> sizes;
```

Comment:
`std::vector<int64> sizes(&t.shape[0], &t.shape[t.dimensions])` will get you a copy of `t.shape` in `std::vector`